atd-vzv/src/views/content/Content.js

```diff
@@ -13,7 +13,7 @@ const Content = () => {
   const routeResult = useRoutes(routes);
   const currentPath = usePath();
 
-  // TODO: Adjust left of content when sidebar opens on mobile
+  // TODO: Slide content to the right when SideDrawer opens
```

Comment:
I just clarified my old comment here because I had to think a bit to figure out what I was saying.

Reply:
self-reflection, always good.